examples/tcp-listener/tcp_listener.bal

```diff
@@ -0,0 +1,44 @@
+// This is the server implementation for the TCP socket.
+import ballerina/io;
+import ballerina/log;
+import ballerina/tcp;
+
+// Bind the service to the port. 
+service on new tcp:Listener(3000) {
+
+    // This remote method is invoked when the new client joins the server.
+    remote function onConnect(tcp:Caller caller) returns tcp:ConnectionService {
+        io:println("Client connected to echoServer: ", caller.remotePort);
+        return new EchoService(caller);
+    }
+}
+
+service class EchoService {
+    tcp:Caller caller;
+
+    public function init(tcp:Caller c) {self.caller = c;}
+
+    // This remote method is invoked once the content is received from the client.
+    remote function onBytes(readonly & byte[] data) returns tcp:Error? {
+        io:println("Echo: ", getString(data));
```

Comment:
For here also, shall we use "string.fromBytes"?

Reply:
updated